Gemfile.lock

```diff
@@ -481,9 +485,9 @@ GEM
       net-ssh (>= 2.8.0)
     sshkit-interactive (0.3.0)
       sshkit (~> 1.12)
-    standard (0.4.2)
-      rubocop (~> 0.83.0)
-      rubocop-performance (~> 1.5.2)
+    standard (0.5.1)
```

Comment:
This is required to get our local version of standard to match what the GH action pulls down.